02-peer-protocol.md

```diff
@@ -580,10 +580,10 @@ shutdown starts, the question of how to behave if it wasn't is avoided:
 the sender always sends a `commitment_signed` first.
 
 As shutdown implies a desire to terminate, it implies that no new
-HTLCs will be added or accepted.  Once any HTLCs are cleared, the peer
+HTLCs will be added or accepted.  Once any HTLCs are cleared, there are no dangling commitments and all updates are included on both commitment transactions, the peer
```

Comment:
Instead of using the word "dangling" can we specify what we mean clearly - something about commitments for which no `revoke_and_ack` has been sent?

Also, nit: please wrap lines at 80 chars, or so.